src/ophys_etl/utils/traces.py

```diff
@@ -1,56 +1,90 @@
 from typing import List
-from functools import partial
 import numpy as np
 from scipy.sparse import coo_matrix
 from scipy.ndimage.filters import median_filter
 
-# Partial for simplifying repeat median filter calls
-medfilt = partial(median_filter, mode='constant')
 
-
-def robust_std(x: np.ndarray) -> float:
+def nanmedian_filter(input_arr: np.ndarray, filter_length: int) -> np.array:
+    """ 1D median filtering with np.nanmedian
+    Parameters
+    ----------
+    input_arr: np.ndarray
+        1d array of signal
+    filter_length: int
+        Length of the median filter to compute a rolling baseline
+
+    Return
+    ------
+    filtered_trace
+    """
+    def fill_nan(input_arr: np.ndarray) -> np.ndarray: 
```

Comment:
Looks good. I would  say make this it's own function in this file and give it a full docstring. Other than that, looks good.

Reply:
Any particular reason to make it into its own function if it's only used specifically for the nanmedian_filter?

Reply:
I think it's cleaner if it's its own function. I found myself uncertain of where the function was when I first looked through the new code.